tests/nosetests/pyanaconda_tests/network_test.py

```diff
@@ -19,9 +19,11 @@
 from pyanaconda import network
 import unittest
 
+
 class NetworkTests(unittest.TestCase):
 
     def is_valid_hostname_test(self):
+        """Test hostname validation."""
```

Comment:
We don't have any convention for this. So I guess @rvykydal should tell if this is fine because he is looking onto this code most of the time.

Reply:
For the record, I'm totally fine with dropping the commit, if having human readable names on tests is not something we need.